scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+| Data output file formats | File formats in which outputs are provided | [VCF](https://gatk.broadinstitute.org/hc/en-us/articles/360035531692) and associated index files; TXT (manifest) |
+
+## Setup
+
+### Getting started with Terra
+
+The GVS workflow is available on Terra, a cloud-based platform for biomedical research. After registering, you can access the [GVS workspace](LINK), which contains instructions for performing joint calling with the Broad GVS using example data uploaded to the workspace. For more information on using the Terra platform, see the [Terra Support Center](https://support.terra.bio/hc/en-us).
+
+If you are new to Terra, you’ll need to [register for a Terra account](https://support.terra.bio/hc/en-us/articles/360028235911). Beta testers of the GVS will be given access to a Broad-managed billing project, but to use Terra for other projects, you’ll need to [set up a billing project](https://support.terra.bio/hc/en-us/articles/360046295092).
+
+The GVS beta workspace in Terra is read-only, so you’ll need to clone the workspace to create a copy where you can upload your own data and run the workflow. For instructions on how to clone workspaces in Terra, see the article [Make your own project workspace](https://support.terra.bio/hc/en-us/articles/360026130851). For a tour of workspaces in Terra, see [Working with workspaces](https://support.terra.bio/hc/en-us/articles/360024743371).
+
+### Workflow requirements
+
+#### Set up BigQuery
+
+To start using the GVS workflow in Terra, you need to create a BigQuery dataset with permissions that will allow Terra to access the dataset by following the instructions below:
+
+**1.** First, **create a dataset in BigQuery** by following the instructions in [Creating datasets](https://cloud.google.com/bigquery/docs/datasets) from the BigQuery documentation.
+
+**2.** Find your Terra proxy group by **selecting the main menu** (three horizontal line icon) at the top left of any Terra page. Next, **click on your account name**, followed by **Profile**. Here, you should see a field called **Proxy Group**, which lists your proxy group underneath. For more information on proxy groups in Terra, see [Pet service accounts and proxy groups](https://support.terra.bio/hc/en-us/articles/360031023592). 
+
+**3.** Grant your proxy group the **BigQuery Data Editor** role on the dataset you created in Step 1 by following the instructions in [Controlling access to datasets](https://cloud.google.com/bigquery/docs/dataset-access-controls) from the BigQuery documentation.
+
+**4.** Grant your proxy group the **BigQuery Data Editor**, **BigQuery Job User**, and **BigQuery Read Session User** roles on the Google Project that contains the dataset you created in Step 1 by following the instructions in [Manage access to projects, folders, and organizations](https://cloud.google.com/iam/docs/granting-changing-revoking-access) from the Google Cloud documentation.
```

Comment:
Is there anything they should see after their done following these steps? Anything that would tell them that they did it right?

Reply:
I added a bit of text to the Quickstart document to describe how users can see if they've setup the permissions correctly.